src/screens/Events/EventsMap.js

```diff
@@ -72,6 +83,12 @@ export default class EventsMap extends Component {
     }
   }
 
+  getEventLabel(eventType) {
+    return types.find((type)=> {
+      return type.value == eventType;
+    }).label;
+  }
```

Comment:
I also think this function could become a utility, or perhaps an Event presenter object.

Reply:
I've never used the presenter pattern but can be easily convinced to do so. I might check once we set Redux up if we can do it in a reducer, though.